test/e2e/test_tools.py

```diff
@@ -197,7 +197,7 @@ def test_xml_upload(self) -> None:
         self.assertTrue(result)
 
         mapping_file = ''
-        for mapping in [x for x in os.scandir('.') if x.name.startswith('id2iri_test-data_mapping_')]:
+        for mapping in [x for x in os.scandir('.') if x.name.startswith('id2iri_test-data-systematic_mapping_')]:
```

Comment:
this could be done a bit more neatly with a `for mapping in glob.glob('id2iri_test-data-systematic_mapping_*'):` (or something along those lines)